### ISSUE TITLE
ESP8266 Large Files

### ISSUE THREAD

**Issue by mue-barakat** [nimaltd/ESP8266#5]
Hello again,

i'm using your driver, it's working perfectly, thanks for that.
But i was wondering how to make the Wifi_RxCallBack read response that is much bigger than _WIFI_RX_FOR_DATA_SIZE 
In another words, i'm trying to get a large JSON object (~30KB), so the server sends that data as chunks
Transfer-Encoding →chunked 
but the ESP never send an interrupt to the STM32

**Comment by nimaltd** [comment]
Hello.  You must split that file.

**Comment by KavehMbeygi** [comment]
Hello , thanks for that library ! but I can`t  set up esp8266 with stm32f103c8t6 ! it dose`not have an example?! thanks for your help...

**Comment by nimaltd** [comment]
No. I have not . This library is too old. I use atc_v2 library.

**Comment by KavehMbeygi** [comment]
hello . in Multiple Connections as TCP Server with ESP8266 , is a timeout mechanism when ESP8266 working as a TCP server. If a TCP client has connected to the ESP8266 TCP server, but there is no data transmission for a period of time, then the server will stop the connection when timeout!
how can i solve this problem with using this library?!

**Comment by nimaltd** [comment]
Ohh. I am sorry . I can not help.
I use arduino to create new firmware.
